components/cd-byline/cd-byline.css

```diff
@@ -0,0 +1,41 @@
+.cd-byline {
+  margin-top: 2rem;/* same as pseudo's padding-bottom */
```

Comment:
I feel like a number with comment that it matches another number should be turned into a custom prop.